### PR TITLE
Update document to reflect new CentOS repo servers

### PR DESCRIPTION
**Pull request by cantlep**
Similar to the other updates but a direct edit on the document this time. References new CentOS repo servers (which are the same as the RHUI CDS') and already in place.

### CODE REVIEW

File: articles/shared/shared-how-access-centos-update-servers.md [hunk 1]
```diff
@@ -19,117 +19,168 @@ toc_mdlink: shared-how-access-centos-update-servers.md
...
 ### Option 1. UKCloud-managed repository servers
 
-We provide repositories and EPEL (Extra Packages for Enterprise Linux) for CentOS 6 and 7 on our Elevated OFFICIAL security domain.
+We provide both CentOS standard repositories and EPEL (Extra Packages for Enterprise Linux) for CentOS 6, 7  and 8 on our Elevated OFFICIAL security domain.
 
-To access CentOS base files, create a file called `centos_<x>_il3.repo` in `/etc/yum.repos.d`, where `<x>` is either 6 or 7 depending on your version of CentOS, and populate it with the following:
+>[!NOTE]
+>Please be aware CentOS6.x is deprecated! You can view details on the CentOS Public Mirror Site [here](http://mirror.centos.org/centos/6.10/readme)
+
+For both Assured and Elevated OFFICIAL security domains, you can configure your systems to reference the following server `https://rh-cds.ukcloud.com` In Assured, this domain is configured in DNS so you will automatically be able to resolve it. For Elevated, you will need to UKCloud Support to get the correct IP address and add an entry to `/etc/hosts` on your systems to be able to resolve it.
```
Comment (mikepukcloud) [inline articles/shared/shared-how-access-centos-update-servers.md:43]: Line 43, missing "Contact" at * in "between you will need to * UKCloud Support"

**Review by mikepukcloud — CHANGES_REQUESTED** [review]
I added a comment on line 43 regarding a missing word (contact)

i woudl have added but not sure i can.

**Review by suehighmoor — APPROVED** [review]
LGTM

File: articles/shared/shared-how-access-centos-update-servers.md (hunk 1 above)
Comment (suehighmoor) [reply articles/shared/shared-how-access-centos-update-servers.md]: @mikepukcloud - fixed this

**Review by mikepukcloud — APPROVED** [review]
LGTM

**Pull request merged**